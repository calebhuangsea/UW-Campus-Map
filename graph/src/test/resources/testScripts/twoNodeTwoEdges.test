# Test add two edges in the same direction between two nodes file
# Tests that a graph can be created
# two nodes can be added to the graph
# two edges can be added in the same direction between 2 nodes

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Add two edges in the same direction
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1