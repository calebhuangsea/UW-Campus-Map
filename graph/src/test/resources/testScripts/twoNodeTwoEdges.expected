# Test add two edges in the same direction between two nodes file
# Tests that a graph can be created
# two nodes can be added to the graph
# two edges can be added in the same direction between 2 nodes

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

# Add two edges in the same direction
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1) n2(e2)